create_framework/separation_of_concerns.rst

```diff
@@ -13,7 +13,7 @@ simple principle: the logic is about creating the Response associated with a
 Request.
 
 Let's create our very own namespace for our framework: ``Simplex``. Move the
-request handling logic into its own ``Simplex\\Framework`` class::
+request handling logic into its own ``Simplex\Framework`` class::
```

Comment:
When using 2 backticks, 1 backslash is enough, otherwise it looks like this:
<img width="789" alt="Screenshot 2019-06-18 10 08 44" src="https://user-images.githubusercontent.com/995707/59664321-193a4980-91b1-11e9-8852-c6d862f64c14.png">

Which isn't a valid namespace.